docker-compose.yml

```diff
@@ -0,0 +1,49 @@
+version: '2'
```

Comment:
What are your thoughts on version 3? Any reason why we shouldn't use it?

Reply:
Found [this](https://vsupalov.com/upgrade-docker-compose-file-version/), and I think keeping in-sync with upstream would be good

Reply:
Thank you for the link to the blog post! 🙇 

Sounds like there aren't many changes to the format and we don't rely on version 2 specific features. I suggest we use version 3 here and don't build up technical debt. The upstream [docker-compose.yml doesn't see many changes](https://github.com/getredash/redash/commits/master/docker-compose.yml) and we have to manually review and apply changes here either way.